pheno_utils/pheno_loader.py

```diff
@@ -544,18 +549,33 @@ def __concat__(self, df1, df2, keep_undefined_research_stage=False):
             self.is_value_in_index(df2, 'undefined', 'research_stage') and not keep_undefined_research_stage:
         
             warnings.warn('filtering "undefined" research_stage..')
-            df = self.join_and_filter_undefined_research_stage(df1, df2)
+            df = self.join_and_filter_undefined_research_stage(df1, df2, how, lsuffix='', rsuffix='')
```

Comment:
just checking: so here suffixes are always kept empty, no matter what the input args say?

Reply:
My mistake, thanks. 
I think that we shouldn't pass parameters here in our use cases, 
but also I shouldn't prevent it (for future use cases).